app/build.gradle.kts

```diff
@@ -162,10 +146,13 @@ dependencies {
     androidTestImplementation(libs.screengrab)
     androidTestImplementation(libs.test.kotlin.coroutines)
 
+    // Jetpack Compose dependencies
+    implementation("androidx.compose.ui:ui:1.4.3")
+    implementation("androidx.compose.material:material:1.4.3")
+    implementation("androidx.compose.ui:ui-tooling:1.4.3")
```

Comment:
Could you add this to the `gradle/libs.versions.toml`?

Reply:
Ok will do this in another commit.